src/ansys/meshing/prime/core/mesh.py

```diff
@@ -0,0 +1,580 @@
+
+    Parameters
+    ----------
+    id : int
+        ID of the mesh.
+    part_id : int
+        ID of the part.
+    part_name : str
+        Name of the part.
+    zone_id : int
+        ID of the zone.
+    zone_name : str
+        Name of the zone.
+    display_mesh_type : DisplayMeshType
+        Type of mesh to display.
+    """
+
+    def __init__(
+        self,
+        id=0,
+        part_id=0,
+        part_name=None,
+        zone_id=0,
+        zone_name=None,
+        display_mesh_type=DisplayMeshType.FACEZONELET,
+    ) -> None:
+        """Initialize display mesh info."""
+        self.id = id
+        self.part_id = part_id
+        self.zone_id = zone_id
+        self.part_name = part_name
+        self.zone_name = zone_name
+        self.display_mesh_type = display_mesh_type
+
+
+def compute_distance(point1, point2) -> float:
+    """Compute the distance between two points.
+
+    Parameters
+    ----------
+    point1 : list
+        List with the coordinates of the first point.
+    point2 : list
+        List with the coordinates of the second point.
+
+    Returns
+    -------
+    float
+        Distance between the two points.
+    """
+    dist = np.linalg.norm(np.array(point2) - np.array(point1))
+    return dist
+
+
+def compute_face_list_from_structured_nodes(dim):
+    """Compute the distances from the nodes.
+
+    Parameters
+    ----------
+    dim : List[int]
+        List with the number of elements in each dimension.
+
+    Returns
+    -------
+    List
+        List with the faces.
+    """
+    flist = []
+    for w in range(dim[2]):
+        for u in range(dim[0] - 1):
+            for v in range(dim[1] - 1):
+                flist.append(4)
+                flist.append(u + v * dim[0] + w * dim[0] * dim[1])
+                flist.append(u + 1 + v * dim[0] + w * dim[0] * dim[1])
+                flist.append(u + 1 + (v + 1) * dim[0] + w * dim[0] * dim[1])
+                flist.append(u + (v + 1) * dim[0] + w * dim[0] * dim[1])
+
+    for v in range(dim[1]):
+        for u in range(dim[0] - 1):
+            for w in range(dim[2] - 1):
+                flist.append(4)
+                flist.append(u + v * dim[0] + w * dim[0] * dim[1])
+                flist.append(u + 1 + v * dim[0] + w * dim[0] * dim[1])
+                flist.append(u + 1 + v * dim[0] + (w + 1) * dim[0] * dim[1])
+                flist.append(u + v * dim[0] + (w + 1) * dim[0] * dim[1])
+
+    for u in range(dim[0]):
+        for v in range(dim[1] - 1):
+            for w in range(dim[2] - 1):
+                flist.append(4)
+                flist.append(u + v * dim[0] + w * dim[0] * dim[1])
+                flist.append(u + (v + 1) * dim[0] + w * dim[0] * dim[1])
+                flist.append(u + (v + 1) * dim[0] + (w + 1) * dim[0] * dim[1])
+                flist.append(u + v * dim[0] + (w + 1) * dim[0] * dim[1])
+    return flist
+
+
+class Mesh(MeshInfo):
+    """Process the mesh for visualization in the GUI.
```

Comment:
```suggestion
    """Processes the mesh for visualization in the GUI.
```
Class descriptions start with verbs ending in "s" or "es"